### PR TITLE
add bootstrap/validation/etc.. scripts [WIP]

### PR DESCRIPTION
**Pull request by epcim**
need a location to store these files, we probably don't want individual repos, as well as we don't want to have multiple copies in each reclass-model/formula etc..

### CODE REVIEW

File: scripts/check_jinja_recursive.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import sys
+import os
+from jinja2 import Environment
+
+def fileList(path, fileTypes):
+  matches = []
+  for root, dirnames, filenames in os.walk(path):
```
Comment (kbaikov) [inline scripts/check_jinja_recursive.py]: Can you please make it pep8 compatible.

**Review by kbaikov — CHANGES_REQUESTED** [review]
Can you please use pep8 for py files and bashate or some other bash linter for sh files?

File: scripts/master_boostrap.sh [hunk 2]
```diff
@@ -0,0 +1,428 @@
+#!/bin/bash -e
```
Comment (fpytloun) [inline scripts/master_boostrap.sh]: This should probably go to `deploy/scripts` directory instead. And there's already some `setup-salt-master-pkg.sh`. This should be unified.

File: scripts/master_init.sh [hunk 3]
```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env bash
```
Comment (fpytloun) [inline scripts/master_init.sh]: I think this also belongs to `deploy/scripts` and should be also unified with what's in there now. There's suddenly a lot of shell scripts..

File: scripts/salt_state_apply_trend.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# source: https://gist.github.com/epcim/e488af8be69b616eab3088a14a937e14
```
Comment (fpytloun) [inline scripts/salt_state_apply_trend.sh]: Description would be really helpful. And maybe also belongs to `deploy/scripts`

**Pull request merged**